packages/dx-react-grid-material-ui/src/templates/drag-drop.test.jsx

```diff
@@ -1,11 +1,19 @@
 import React from 'react';
 import { Paper } from 'material-ui';
-import { mountWithStyles } from '../utils/testing';
+import { createMount, getClasses } from 'material-ui/test-utils';
 import { Container, styleSheet } from './drag-drop';
 
 describe('Container', () => {
+  let mount;
+  beforeAll(() => {
```

Comment:
It seems strange to setup and clear mount once. This stuff should run each test case according to [the docs](https://material-ui-1dab0.firebaseapp.com/guides/testing#createmount-options-mount).